ghost/core/core/frontend/helpers/split.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function split(...args) {
+    const options = args.pop();
+    const data = options.data || {};
+    const separator = options.hash.separator !== undefined ? options.hash.separator : ',';
```

Comment:
Can be shortened to 
`const separator = options.hash.separator || ',';` 

Same as the line above and in the concat helper :)